### DISCUSSION TITLE
Plot quality check from the GUI

### DISCUSSION THREAD

**Discussion by brianwfreeman** [Chia-Network/chia-blockchain-gui#2380] [Ideas]
It would be nice if the GUI had a way for users to check the quality of their plots. Right now the only way to do it is from command line.

**Comment by parkroyk** [comment]
It would be great if we can add the following features to the plotter itself:

1. An option to specify plot check with -check <n>
2. An option to specify minimum PASS quality score and replots if it doesn't pass

**Comment by BrandtH22** [comment]
@brianwfreeman , note I transferred this enhancement request to the GUI repo for better visibility 👍
